charts/artifact-hub/values.schema.json

```diff
@@ -360,6 +360,11 @@
                             "type": "boolean",
                             "default": true
                         },
+                        "defaultBackendEnabled": {
```

Comment:
Can you add the new field to the list of required fields in the ingress object?

https://github.com/artifacthub/hub/blob/ceb8b9d8c434424e08c326dbf3c5d8ab1159068b/charts/artifact-hub/values.schema.json#L382-L385